src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -143,6 +191,24 @@ export default function AttachmentsOverview() {
             onChange={e => handleUpload(e, e.target.files)}
           />
         </form>
+        <div className={styles.cameraSection}>
+          <button className="cameraButton" onClick={openCamera}>
+            Camera
+          </button>
+          {cameraIsOpen && (
```

Comment:
I think this whole thing should be moved to a new component. Ideally, the whole state related to camera is in this component.

Something like `<Camera />`. As props we'd have something like `onNewImage`, which is when a camera component informs us that a new image has been successfully uploaded and should be added to the gallery.

Would that make sense?

Reply:
This makes sense. But do we have to name the component Camera? I mean I library already provides a component with that name. Will this act as a wrapper?

Reply:
Well, first of all its about the reusability discussed last time, but second it should bring the lib into the business domain of OpenMRS. As such the API communication for upload should be part of that (as described).

The name was freely chosen. If you'd see a better name (maybe `CameraUpload`?) feel free to choose one.